### DISCUSSION TITLE
A message about Tweeten's future

### DISCUSSION THREAD

**Discussion by MehediH** [MehediH/Tweeten#327] [Announcements]
Hi all,

I have received a lot of messages about Tweeten not working or being broken in some sort of way over the last few days. Twitter's new leadership have made it very clear that they do not care about third-party developers like me, and unfortunately, I can't afford to spend more time on a project that can be made obsolete by a billionaire overnight.

As a result, Tweeten will no longer receive any updates going forward. This is a hard decision on my side, as Tweeten has been my biggest project to date. I hope you all understand :)

I am very grateful for all the support from everyone over the last few years. Please feel free to stop sponsoring me / donating on Patreon, etc., as I will not be working on Tweeten going forward. Thank you for all the support.

If you want to follow my work going forward, [please check out SIEN](https://twitter.com/sienvision), where a couple of my friends and I will be launching some cool stuff. You can also [follow me on Twitter here.](https://twitter.com/mehedih_)

Thank you,
Mehedi

**Comment by KonradIT** [comment]
Fully understand your decision and thanks for the hard work over the years. Better Tweetdeck extension, also deprecated, works for the time being.

**Reply by jaydeflix** [reply]
You did good work.

**Comment by dna75** [comment]
Thanks for all the effort! I understand your point of view.

**Comment by DatDraggy** [comment]
I've been using Tweeten for such a long time that I don't even remember when I started. Thank you so much for the work you've done, it's been a great tool <3
Totally understand, there's no point putting any more work into this sinking ship. Sucks how Mr Elon Muskyhusky is running this thing into the ground...

**Comment by ferrazpedro** [comment]
By any chance, would you open source tweeten's code? I think it would be awesome since this was the best TweetDeck client to date.

**Reply by theguly** [reply]
this. totally see your point but probably other could take the ownership and maintain the code, until Dogewitter restricts the API even more.

**Comment by joaociocca** [comment]
That's really sad news on one side. Expected, but sad. On the other, I'm deeply thankfull for the amazing experience - even now, at least to me, Tweeten keeps working awesomely. Thanks for all these years of a wonderful experience, @MehediH !!

**Comment by cjeon** [comment]
Really loved your project. Thank you for your contribution to the community. Wish you a good luck!

**Comment by Luke717** [comment]
Hi @MehediH  . I started using Tweeten in its early days. I contributed £ to its early development and I've been using it every day on Windows PCs since then. Thank you for developing and supporting it, unquestionably the best third-party Twitter essential.

I think it's finally stopping working properly. It still loads and updates timelines but over the past week, I've seen constant mismatches with notifications where Tweeten doesn't show them in a timely way, or even show all of them, but they are very timely on the official Twitter app and on the web. It seems the API tap has moved.

Someone yesterday tweeted a suggestion to edit the cookies file in a SQL editor and add 'legacy' to the line 'tweetdeck_version' and this person reported success. I checked my file and saw this line already has 'legacy' as the filter. Here's the relevant thread with screenshots: https://twitter.com/jangles/status/1653357752717131778

Is there another way to achieve the desired result? Or are we really at the end of the road now?

Thanks.

**Comment by damianh** [comment]
@MehediH This app made twitter usable for me, thank you for sharing this project with us all. 

Have you considered making a version that supports BlueSky? If not would you consider opensourcing it and maybe someone (e.g. me) will do the work to make it Bluesky compatible?
